app/src/main/java/com/gathering/android/profile/ProfileComposables.kt

```diff
@@ -0,0 +1,126 @@
+package com.gathering.android.profile
```

Comment:
why do we need a new file for this ? are these compassables going to be reused later ?

Reply:
yes the one here is possibly going to be used later but also it makes code look cleaner and easier to read.